add doppel-cli recipe

jameslamb opened the pull request:
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/doppel-cli```) and found some lint.

Here's what I've got...


For **recipes/doppel-cli**:

 * Failed to even lint the recipe, probably because of a conda-smithy bug :cry:. This likely indicates a problem in your `meta.yaml`, though. To get a traceback to help figure out what's going on, install conda-smithy and run `conda smithy recipe-lint .` from the recipe directory.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/doppel-cli```) and found it was in an excellent condition.

jameslamb commented:
@conda-forge-admin, please ping team

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I was asked to ping @conda-forge/staged-recipes and so here I am doing that.

xhochy commented on recipes/doppel-cli/bld.bat:
```diff
@@ -0,0 +1,2 @@
+@echo on
```
Doesn't using `pip` not work here?

xhochy commented on recipes/doppel-cli/meta.yaml:
```diff
@@ -0,0 +1,43 @@
+    - python
+    - tabulate
+test:
+  requires:
```
`pytest` is not required for the commands? Please remove.

xhochy commented on recipes/doppel-cli/meta.yaml:
```diff
@@ -0,0 +1,43 @@
+  url: "https://pypi.io/packages/source/d/doppel-cli/doppel-cli-{{ version }}.tar.gz"
+  sha256: ea363bba25f6f38f64e5a3a9d52d11e01483c1c532eb6bcd85eabbc89118cd1a
+build:
+  number: 0
```
Seems like this can be `noarch: python`.

xhochy commented:
Might be helpful to generate this recipe using https://conda-forge.org/blog//2020/03/05/grayskull

jameslamb commented:
> Might be helpful to generate this recipe using https://conda-forge.org/blog//2020/03/05/grayskull

hey thanks @xhochy ! I wish I knew about `grayskull` before, it's really great. Would have save me a lot of time.

On https://github.com/conda-forge/staged-recipes/commit/ae6dd9021ba5d273dbf3bb7bdfb55e337715abde I checked in the meta file generated by `grayskull pypi doppel-cli`. Seems like it addresses all of your comments
